ArmaforcesMissionBot/Modules/Signups.cs

```diff
@@ -657,6 +657,40 @@ public async Task EditMission(int missionNo)
                 }
             }
         }
+		
+        [Command("edytuj-misje")]
+        [Summary("Po podaniu nazwy misji jako parametru włączy edycje danej misji (część bez zespołów).")]
+        [ContextDMOrChannel]
+        public async Task EditMission([Remainder] string title)
```

Comment:
I think it would be much better to create this overload:
```suggestion
        public async Task EditMission(IGuildChannel channel)
```
It will be much easier to use.

No need for the `string` overload if you don't want to allow specifying only part of the mission name e.g. `AF!edytuj-misje Mission Tit` for the mission titled `Mission Title`. I guess that's not needed with channel mentions and Discord suggestions in place.